### PR TITLE
[EWS] Remove ChangeLog steps

### PR DESCRIPTION
**Pull request by JonWBedard**
#### 452ee7982845475da6114437f62e77ef79e8cf64
<pre>
[EWS] Remove ChangeLog steps
<a href="https://bugs.webkit.org/show_bug.cgi?id=240542">https://bugs.webkit.org/show_bug.cgi?id=240542</a>
&lt;rdar://problem/93456184 &gt;

Reviewed by Aakash Jain.

* Tools/CISupport/ews-build/factories.py:
* Tools/CISupport/ews-build/steps.py:
(ValidateCommitterAndReviewer.start):
(SetBuildSummary.start):
(ValidateChangeLogAndReviewer): Deleted.
(FindModifiedChangeLogs): Deleted.
(CreateLocalGITCommit): Deleted.
* Tools/CISupport/ews-build/steps_unittest.py:

Canonical link: <a href="https://commits.webkit.org/250800@main">https://commits.webkit.org/250800@main</a>
git-svn-id: <a href="https://svn.webkit.org/repository/webkit/trunk@294545">https://svn.webkit.org/repository/webkit/trunk@294545</a> 268f45cc-cd09-0410-ab3c-d52691b4dbfc
</pre>

### CODE REVIEW

File: Tools/CISupport/ews-build/steps.py [hunk 1]
```diff
@@ -1680,39 +1680,6 @@ def start(self):
         return None
 
 
-class ValidateChangeLogAndReviewer(shell.ShellCommand):
-    name = 'validate-changelog-and-reviewer'
-    descriptionDone = ['Validated ChangeLog and Reviewer']
-    command = ['python3', 'Tools/Scripts/webkit-patch', 'validate-changelog', '--check-oops', '--non-interactive']
```
Comment (aj062) [inline Tools/CISupport/ews-build/steps.py]: without this how is Merge-Queue validating reviewer for a PR which contains "Reviewed by XYZ" line in commit message and doesn't have explicit approval in GitHub UI?

Comment (JonWBedard) [reply Tools/CISupport/ews-build/steps.py]: [class ValidateCommitMessage(steps.ShellSequence, ShellMixin):](https://github.com/WebKit/WebKit/blob/622f5bf389dec8cff2e4bf744b1e17a15c751edd/Tools/CISupport/ews-build/steps.py#L5004), although it looks like we don't check that the listed reviewer is actually a reviewer, so we need a step that does that.

**Review by aj062 — APPROVED** [review]
r+ as Jonathan mentioned that the Reviewer check would be done in a separate PR.

File: Tools/CISupport/ews-build/steps.py (hunk 1 above)
Comment (JonWBedard) [reply Tools/CISupport/ews-build/steps.py:1686]: Oh yikes, I totally got this one wrong! I don't know why I thought `ValidateCommitterAndReviewer` used changelogs, but it doesn't We definitely need that step! I just need to have `ValidateCommitMessage` populate the `reviewer` field.

Comment (JonWBedard) [reply Tools/CISupport/ews-build/steps.py]: To @aj062's point, https://bugs.webkit.org/show_bug.cgi?id=240718 will cover adding a check for reviewer status in the `ValidateCommitMessage` step.

**Pull request merged**